### ISSUE TITLE
Info about the project in the README.md

### ISSUE THREAD

**Issue by bcipolli** [scitran/sdm#58]
The `README.md` is empty, so I don't know what this project is really about, how to use it, or even what it looks like... :-/

**Comment by wandell** [comment]
Sorry.

http://post.stanford.edu


On Nov 28, 2015, at 2:50 PM, Ben Cipollini <notifications@github.com> wrote:


The README.md is empty, so I don't know what this project is really about, how to use it, or even what it looks like... :-/

—
Reply to this email directly or view it on GitHub<https://github.com/scitran/sdm/issues/58>.

———
Brian A. Wandell
Stein Family Professor
Cognitive and Neurobiological Imaging (CNI)
Psychology Department
Phone: 1-650-725-2466
http://www.stanford.edu/~wandell

**Comment by kofalt** [comment]
Several of our component repos are sadly lacking readmes at this juncture - for now, you can check [scitran/scitran](https://github.com/scitran/scitran) for information :)

@rentzso - feel like taking this one on? A quick blurb about the details & layout of the web app?

**Comment by rentzso** [comment]
Yes, we are going to work on documentation in the upcoming weeks.
